roles/cluster_logging/files/kibana/provision-index.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/sh
+
+#
+# Script to setup the default index pattern in kibana
+#
+
+INDEX_ID="logstash-*"
```

Comment:
Consider moving this outside of the script, into the job definition, and have an assertion it's set here.